python/rikai/spark/functions/vision.py

```diff
@@ -113,6 +120,8 @@ def video_to_images(
     sample_rate: int = 1,
     max_samples: int = 15000,
     quality: str = "worst",
+    img_format: str = "png",
```

Comment:
should we either name this as `format` or `image_format`?

while `format` is reserved word, it is not worse than what PIL takes ?

Reply:
Additionally, can we test them in the `pyspark.DataFrame` ,and in `spark sql` format?

I'd like to see the actual UX spark sql.

Reply:
naming: i wanted to distinguish the video options from the image options. Here there are many parameters so it's easy for users to get confused.

in DataFrame format with UDF you can safely omit the arguments for optional parameters. If you need to specify these then it's just adding `img_format=lit("png")` no?

SparkSQL - i can check but i don't think it's a problem either?

Reply:
wrt `image_format` I guess I prefer brevity where the abbreviation is obvious. I can change it to that given it might not be obvious to all.